Watchman.Discord/Areas/Responses/Controllers/ResponsesController.cs

```diff
@@ -36,83 +27,83 @@ public class ResponsesController : IController
         [DiscordCommand("add response")]
         public async Task AddResponse(DiscordRequest request, Contexts contexts)
         {
-            var messageService = _messagesServiceFactory.Create(contexts);
+            var messageService = this._messagesServiceFactory.Create(contexts);
             var onEvent = request.Arguments.FirstOrDefault(x => x.Name?.ToLowerInvariant() == "onevent")?.Value;
             var message = request.Arguments.FirstOrDefault(x => x.Name?.ToLowerInvariant() == "message")?.Value;
             if (onEvent == null || message == null)
             {
-                await messageService.SendResponse(x => x.NotEnoughArguments(), contexts);
+                await messageService.SendResponse(x => x.NotEnoughArguments());
                 return;
             }
-            var response = await _responsesService.GetResponseByOnEvent(onEvent);
+            var response = await this._responsesService.GetResponseByOnEvent(onEvent);
```

Comment:
dlaczego usuwasz contexts? skąd teraz będziemy brać dane? (w skrócie)
mam tu na myśli dane takie jak użytkownik, kanał itd

Reply:
stąd: var messageService = this._messagesServiceFactory.Create(contexts);

Reply:
widze